log/logger.go

```diff
@@ -0,0 +1,52 @@
+package log
```

Comment:
I don't think we should name this the same as the standard library's `log` package.

https://go.dev/blog/package-names
`Avoid unnecessary package name collisions. While packages in different directories may have the same name, packages that are frequently used together should have distinct names. This reduces confusion and the need for local renaming in client code. For the same reason, avoid using the same name as popular standard packages like io or http.`

But then I don't know what it should be called 😅